### PR TITLE
"Added Footer section"

### PR DESCRIPTION
**Pull request by falgun143**
Solves issues #29 .Please review it .

### CODE REVIEW

File: src/Components/Footer.jsx [hunk 1]
```diff
@@ -1,16 +1,200 @@
...
+        </div>
+
+        <div style={{ display: "flex", gap: "20px" }}>
+          <Facebook
+        
+           
+            onClick={() => {
+              window.location = "https://www.facebook.com/GDSC.SMIT/";
+            }}>
+            <FaFacebookF style={{ color: "#1A6ED8",marginBottom:"15px" }}></FaFacebookF>
```
Comment (pooranjoyb) [inline src/Components/Footer.jsx:124]: `GDSC-SMIT` isn't in Facebook, please update the same with Instagram.

File: src/Components/Footer.jsx [hunk 2]
```diff
@@ -1,16 +1,200 @@
...
+            <span style={{ color: "white" }}>Facebook</span>
+
+          </Facebook>
+
+          <Youtube
```
Comment (pooranjoyb) [inline src/Components/Footer.jsx:129]: Also, please remove Youtube :/

File: src/Components/Footer.jsx [hunk 3]
```diff
@@ -1,16 +1,200 @@
...
+      </div>
+
+      <div
+        style={{
+          display: "flex",
+          justifyContent: "center",
+          paddingTop: "10px",
+        }}
+      >
+        <h4>© 2023 COPYRIGHT GDSC SMIT</h4>
```
Comment (pooranjoyb) [inline src/Components/Footer.jsx:194]: Reduce the size of this text :)

File: src/Components/Footer.jsx [hunk 4]
```diff
@@ -1,16 +1,200 @@
...
+          alignItems: "center",
+          height: "500px",
+          backgroundColor: "white",
+          color: "black",
+          fontWeight: "bold",
+
+          padding: "5px",
+        }}
+      >
+        <div>
```
Comment (pooranjoyb) [inline src/Components/Footer.jsx:76]: ### Remove these content : 

image
image

**Review by pooranjoyb — CHANGES_REQUESTED** [review]
Please also reduce the size of Brand Icons and make sure they're responsive in smaller screens.

**Comment by falgun143** [comment]
@pooranjoyb Sure ,I will make the changes and give you the update.

**Comment by pooranjoyb** [comment]
> @pooranjoyb Can you give me the link for the instagram and also tell me that .After removing about and help section what should i put there can i do something like this or remove this one also. image

Nope, just make it minimalistic :) @falgun143

**Comment by falgun143** [comment]
@pooranjoyb  please review it.

**Review by pooranjoyb — APPROVED** [review]

**Pull request merged**